db/chores/update_auctions_billable_data.rb

```diff
@@ -0,0 +1,21 @@
+class UpdateAuctionsBillableData
```

Comment:
if we made it relational we would still need to update. Here I am updating just for consistency. Eg: Rather than `"Micropurchase"` in the `billable_to` field we will now have `"Acquisition Services / Marketplaces / Micropurchasing (Billable)"` (what you would select in the form if you created the auction today)